source/extensions/filters/http/ext_authz/ext_authz.cc

```diff
@@ -78,6 +78,16 @@ Http::FilterHeadersStatus Filter::decodeHeaders(Http::RequestHeaderMap& headers,
   skip_check_ = skipCheckForRoute(route);
 
   if (!config_->filterEnabled() || skip_check_) {
+    if (skip_check_) {
+      return Http::FilterHeadersStatus::Continue;
+    }
+    if (config_->denyAtDisable()) {
```

Comment:
Let's do logging in here? So it is easier to debug when a request is rejected.

Reply:
Added!
Thanks so much for helping review the code